h2o-app/src/main/java/water/H2OApp.java

```diff
@@ -1,10 +1,13 @@
 package water;
 
 public class H2OApp extends H2OStarter {
+  
+  public static int BAD_JAVA_VERSION_RETURN_CODE = 3;
```

Comment:
Maybe it would make sense to do enum for all return codes used in code? What do you think? @michalkurka